config/crd/bases/capsule.clastix.io_tenants.yaml

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
Same here, please, keep the empty new line at the beginning if this has been generated by `controller-gen`